src/include/cpu/rpp_cpu_simd.hpp

```diff
@@ -688,6 +733,142 @@ inline void rpp_normalize48(__m128 *p)
 
 // AVX loads and stores
 
+inline void rpp_load48_u8pkd3_to_f32pkd3_avx(Rpp8u *srcPtr, __m256 *p)
+{
+    __m128i px[4];
```

Comment:
It is hard to verify all these AVX intrinsics in the core review. I suggest having a mechanism in the code which verify the AVX code automatically. Basically we need to run reference c code, store output in some temp buffers, run AVX code and compare if it exactly matching the reference c implementation